x-pack/platform/plugins/shared/stack_connectors/public/connector_types/slack_api/slack_connectors.tsx

```diff
@@ -42,152 +34,65 @@ const getSecretsFormSchema = (docLinks: DocLinksStart): SecretsFieldSchema[] =>
   },
 ];
 
-interface AllowedChannels {
-  id: string;
-  name: string;
-}
-
-const getConfigFormSchemaAfterSecrets = ({
-  options,
-  isLoading,
-  isDisabled,
-  onChange,
-  onCreateOption,
-  selectedOptions,
-}: {
-  options: Array<EuiComboBoxOptionOption<AllowedChannels>>;
-  isLoading: boolean;
-  isDisabled: boolean;
-  onChange: (options: Array<EuiComboBoxOptionOption<AllowedChannels>>) => void;
-  onCreateOption: (searchValue: string, options: EuiComboBoxOptionOption[]) => void;
-  selectedOptions: Array<EuiComboBoxOptionOption<AllowedChannels>>;
-}): ConfigFieldSchema[] => [
+const getConfigFormSchema = (): ConfigFieldSchema<string>[] => [
   {
     id: 'allowedChannels',
-    isRequired: true,
+    isRequired: false,
     label: i18n.ALLOWED_CHANNELS,
     type: 'COMBO_BOX',
+    validations: [
+      {
+        isBlocking: true,
+        validator: (args) => {
+          const valueAsArray = Array.isArray(args.value) ? args.value : [args.value];
+
+          if (valueAsArray.length === 0) {
+            return;
+          }
+
+          const areAllValid = valueAsArray.every((value) => value.startsWith('#'));
```

Comment:
does this not accept channel ids anymore? What's the point in forcing # as prefix in the UI then?